src/Components/Loading/Loading.js

```diff
@@ -1,27 +1,16 @@
 import {
     EmptyState,
+    EmptyStateHeader,
     EmptyStateIcon,
-    Title
+    Spinner
 } from '@patternfly/react-core/dist/esm/components';
 
 import React from 'react';
 
-const Loading = () => {
-    const Spinner = () => (
-        <span className="pf-c-spinner" role="progressbar" aria-valuetext="Loading...">
-            <span className="pf-c-spinner__clipper" />
-            <span className="pf-c-spinner__lead-ball" />
-            <span className="pf-c-spinner__tail-ball" />
-        </span>
-    );
-
-    return <EmptyState>
-        <EmptyStateIcon variant="container" component={Spinner} />
-        <Title size="lg" headingLevel="h4">
-            Loading
-        </Title>
-    </EmptyState>;
-
-};
+const Loading = () => (
+    <EmptyState>
+        <EmptyStateHeader titleText="Loading" headingLevel="h4" icon={<EmptyStateIcon icon={Spinner} />} />
+    </EmptyState>
+);
```

Comment:
PF5 supports spinner in empty state natively now, so the code is simpler and spinner is better animated